.golangci.yml

```diff
@@ -11,14 +11,6 @@ linters-settings:
     min-complexity: 20
   goconst:
     ignore-tests: true
-  goheader:
```

Comment:
Oh no, we need the goheader linter. It's removed in the template, because it's meant to be used by _anyone_, but if it's our connector we want to make sure that files contain the copytight notice. Also please add the `.golangci.goheader.template` file, for example you can get it [here](https://github.com/ConduitIO/conduit-connector-s3/blob/main/.golangci.goheader.template).